File: Practical/HTTP_Example/NodeJS_Clients/GET.js
const axios = require('axios')

// Program to make a simple GET request and parse the Response Body
const HOST = "http://localhost:";
const PORT = "3000"
const HTML_RES_PATH = "/"
const JSON_RES_PATH = '/json'
const DB_PATH = '/sensor/value'

// Make a GET request to Server
axios.get(HOST + PORT + DB_PATH)
    .then((response)=>{
        const RESPONSE_BODY  = JSON.stringify(response.data, null,2);
        console.log("Data: "+RESPONSE_BODY+". Response Code: "+ response.status)
    }).catch(error=>{
        console.log(error);
    })

    

File: Practical/HTTP_Example/Python_Clients/GET.py
# Program to perform simple GET request to HTTP server
import requests
import json

HOST = "http://localhost:"
PORT = "3000"
HTML_RES_PATH = "/"
JSON_RES_PATH = "/json"
DB_PATH = "/sensor/value"

# Make HTTP GET Request to Server
response = requests.get(HOST + PORT + DB_PATH)


# Printing the JSON response with inbuilt encoder
print(response.json())
print(response.status_code)